src/platform/plugins/shared/discover/public/application/main/state_management/discover_saved_search_container.ts

```diff
@@ -91,7 +94,7 @@ export interface DiscoverSavedSearchContainer {
    * @param id
    * @param dataView
    */
-  load: (id: string, dataView?: DataView) => Promise<SavedSearch>;
+  load: (id: string) => Promise<SavedSearch>;
```

Comment:
The `dataView` param didn't seem to be referenced anywhere in code, so I removed it.